shell/browser/api/electron_api_system_preferences_win.cc

```diff
@@ -243,10 +243,10 @@ LRESULT CALLBACK SystemPreferences::WndProc(HWND hwnd,
 }
 
 void SystemPreferences::OnSysColorChange() {
-  bool new_invertered_color_scheme = IsInvertedColorScheme();
-  if (new_invertered_color_scheme != invertered_color_scheme_) {
-    invertered_color_scheme_ = new_invertered_color_scheme;
-    Emit("inverted-color-scheme-changed", new_invertered_color_scheme);
+  bool new_inverted_color_scheme = IsInvertedColorScheme();
+  if (new_inverted_color_scheme != inverted_color_scheme_) {
+    inverted_color_scheme_ = new_inverted_color_scheme;
+    Emit("inverted-color-scheme-changed", new_inverted_color_scheme);
```

Comment:
Just a note for future PRs: I'm generally fine with typo fixes in PRs when they deal with comments, docs, and tests. 

Changing actual variables in code as part of a larger typo-fixing PR - when the rest of the PR doesn't directly deal with the corresponding code - is riskier, and I'd really like to avoid doing that. These specific changes seem small and don't negatively impact the rest of the code, so I'm fine merging them with this PR, but this would likely be a merge blocker from me in the future.